src/NUnitFramework/framework/Internal/TestExecutionContext.cs

```diff
@@ -484,9 +484,19 @@ private TestExecutionContext CreateIsolatedContext()
             return context;
         }
 
-#endregion
+        /// <summary>
+        /// Send a test message to listeners
```

Comment:
`Sends` and please add a period.

Should we mention that it goes in the test result XML, or is that overwhelmingly obvious?